third_party/tonic/converter/dart_converter.h

```diff
@@ -415,6 +417,70 @@ struct DartConverter<std::vector<T>> {
   }
 };
 
+/// Json style map.
```

Comment:
I would prefer we avoid this if we can. It's harder to use API like this consistently across multiple platforms/implementations. We'll lose a fair amount of static typechecking here, and it'll be easier to break an implementation unexpectedly.

It's also more expensive/involves a lot more Dart_* api calls than we typically do for argument conversion.

Typically, we use Uint8Lists to pack/unpack structured data between Dart land and native.